### ISSUE TITLE
Truncate and/or delete all rows in a table

### ISSUE THREAD

**Issue by cheapRoc** [jackc/pgx#394]
I'm trying to clear out all rows in a series of test tables prior to running tests. I would imagine I can simply run a `TRUNCATE` and `TRUNCATE * CASCADE` against these tables using something like `ConnPool.Query` or `ConnPool.QueryEx`. However, that doesn't appear to return properly. Sometimes the query function returns without having truncated (I'm guessing that the query hasn't fired in the database yet), other times it appears to deadlock or locks up and never returns from the query function.

Any tips or pointers on how to perform this? I haven't spent much time yet debugging figuring I must be missing something simple here.

I'll happily paste some example code if need be.

Thanks!

**Comment by jackc** [comment]
`Query(Ex)` expects is for queries that return a result set. `Exec(Ex)` is what you want.

**Comment by cheapRoc** [comment]
Thank you, using `Exec(Ex)` definitely helped but did not solve the problem entirely. I'm using CockroachDB and I needed to perform the following as well...

```sql
SET sql_safe_updates = false;
```

The table structure that I had included foreign keys across two tables so when I was using things like `DELETE FROM <table name>` I was receiving an error that instructed me about the `sql_safe_updates` flag (default set to `true`). That error message did not bubble up through `pgx`. Should that be the case?

Otherwise, I'm all set with this issue.
